btrack/_localization.py

```diff
@@ -188,16 +192,17 @@ def segmentation_to_objects(
     # objects
     if "label" in properties:
         logger.warning("Cannot use scikit-image label as a property.")
-        del properties["label"]
+        properties = set(properties)
+        properties.remove("label")
+        properties = tuple(properties)
+        assert "label" not in properties
```

Comment:
Surely this is unnecessary? The fact that you convert to `set` even covers the situation in which `label` appeared twice in `properties`

Reply:
That was a sanity check on my behalf 🙂